pkg/bugzilla/client.go

```diff
@@ -0,0 +1,80 @@
+// Package bugzilla implements a basic bugzilla jsonrpc api wrapper with a single query
+package bugzilla
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// Bugzilla object with information to access the bugzilla api
+type Bugzilla struct {
+	username string
+	password string
+	url      string
+}
+
+// clientRequest provides a basic json request implementation
```

Comment:
blech, this comment is bad.  Be more descriptive, like `clientRequest is the jsonrpc wrapper structure for requests`.